services/ymax-planner/README.md

```diff
@@ -108,6 +108,7 @@ Environment variables:
 - `SPECTRUM_API_TIMEOUT`: Overrides `REQUEST_TIMEOUT` for the Spectrum REST API (optional)
 - `SPECTRUM_API_RETRIES`: Overrides `REQUEST_RETRIES` for the Spectrum REST API (optional)
 - `DOTENV`: Path to environment file containing defaults of above (default ".env")
+- `SQLITE_DB_PATH`: The path where the SQLiteDB used by the resolver should be created. While a relative path can be provided, and absolute path is preferred. 
```

Comment:
DOTENV explicitly refers to the "above" variables, which means it must come last.
```suggestion
- `SQLITE_DB_PATH`: The path where the SQLiteDB used by the resolver should be created. While a relative path can be provided, and absolute path is preferred. 
- `DOTENV`: Path to environment file containing defaults of above (default ".env")
```